rpc/src/lib.rs

```diff
@@ -141,7 +141,11 @@ pub fn init(spawn: impl Spawn + Clone + Send + Sync + 'static) {
 }
 
 pub(crate) fn spawn(future: impl Future<Output = ()> + Send + 'static) -> Result<(), SpawnError> {
-    SPAWN.with(|spawn| spawn.borrow_mut().spawn(future))
+    if SEED_SPAWN.get().is_some() {
```

Comment:
I actually am in favor of getting rid of SEED_SPAWN and init entirely; sorry if that wasn't clear! The `spawn_boxed` looks good and we can just always use that line.

Reply:
Sure we can, but that will make `runtime` mandatory for everyone who wants to use `tarpc`. Whether that is desirable (or not) is a matter of discussion, so I've left this safety hatch for now.

Reply:
Ah, fair point! Let's leave it for now. I have a branch wherein I make all usage of the global executor opt-in, to facilitate the removal of mandatory `Send + 'static` bounds. We can revisit this then.